modules/play25/src/main/scala/be/wegenenverkeer/atomium/play/PlayJsonCodec.scala

```diff
@@ -6,7 +6,7 @@ import play.api.libs.json.{Format, Json}
 /**
   * Created by Karel Maesen, Geovise BVBA on 18/11/16.
   */
-case class PlayJsonCodec[E](implicit val entryFormat: Format[E]) extends FeedPageCodec[E, Array[Byte]] {
+case class PlayJsonCodec[E]()(implicit val entryFormat: Format[E]) extends FeedPageCodec[E, Array[Byte]] {
```

Comment:
I changed the signature because of this warning:

```
[warn] /data/projects/atomium/modules/play25/src/main/scala/be/wegenenverkeer/atomium/play/PlayJsonCodec.scala:9: case classes should have a non-implicit parameter list; adapting to 'case class PlayJsonCodec()(...)'
[warn] case class PlayJsonCodec[E](implicit val entryFormat: Format[E]) extends FeedPageCodec[E, Array[Byte]] {
[warn]                            ^
[warn] /data/projects/atomium/modules/play25/src/main/scala/be/wegenenverkeer/atomium/play/PlayJsonEntryCodec.scala:11: case classes should have a non-implicit parameter list; adapting to 'case class PlayJsonEntryCodec()(...)'
[warn] case class PlayJsonEntryCodec[E](implicit val entryFormat: Format[E]) extends Codec[E, String] {
[warn]                                 ^
```

I suppose it's not a breaking change because the compiler itself is adapting it to 'case class PlayJsonCodec()(...)'